pkg/configuration/configuration.go

```diff
@@ -270,6 +298,26 @@ func (c *Config) GetRegistrationServiceURL() string {
 	return c.host.GetString(varRegistrationServiceURL)
 }
 
+// GetConsoleNamespace returns the console route namespace
+func (c *Config) GetConsoleNamespace() string {
+	return c.host.GetString(varConsoleNamespace)
+}
+
+// GetConsoleRouteName returns the console route name
+func (c *Config) GetConsoleRouteName() string {
+	return c.host.GetString(varConsoleRouteName)
+}
+
+// GetCheNamespace returns the Che route namespace
+func (c *Config) GetCheNamespace() string {
+	return c.host.GetString(varCheNamespace)
+}
+
+// GetCheRouteName returns the name of the Che dashboard route
+func (c *Config) GetCheRouteName() string {
+	return c.host.GetString(varCheRouteName)
+}
```

Comment:
you didn't add tests for these methods

Reply:
added here: fa2316815d63d4a7db87100bc4a77cd7235df3ef

Reply:
and here: a77e5a89d165a6d496d8ef1a52de1a16f5109e0e